pintracker/stateless/stateless.go

```diff
@@ -604,27 +511,27 @@ func (spt *Tracker) localStatus(ctx context.Context, incExtra bool) (map[string]
 		return nil, err
 	}
 
+	pininfos := make(map[string]*api.PinInfo)
```

Comment:
one thing I learnt is that make(map) takes an optional number which allows Go to be more intelligent when pre-allocating the needed memory. If this map will grow to the size of the statePins, then you can use that with make.